src/main/java/org/toradocu/extractor/ThrowsTag.java

```diff
@@ -76,6 +78,31 @@ public void setCondition(String condition) {
 	}
 	
 	/**
+	 * Sets the translated conditions for this throws tags to the given conditions. Each element in
+	 * the set is combined using an || conjunction.
+	 * 
+	 * @param conditions the translated conditions for this throws tag (as Java boolean conditions)
+	 * @throws NullPointerException if conditions is null
+	 */
+	public void setConditions(Set<String> conditions) {
```

Comment:
Is it possible to move the entire logic of this method in the condition translator? In this way we'd keep only `setCondition(String condition)` in this class.